.rspec

```diff
@@ -0,0 +1,2 @@
+--require spec_helper
```

Comment:
Lets remove `--require spec_helper`. Its causing an error when we try to run the tests.

Reply:
Done :) Thank you